001_map_equidist_EQ/GMT_EQglob_BFO.gmt

```diff
@@ -64,7 +64,7 @@ cmap2use=viridis.cpt
 
 
 # inferno_mod.cpt, magma_mod.cpt and plasma_mod.cpt are slightly modified versions of 
-# inferno.cpt, magma.cpt and plasma.cpt that can be also download from: 
```

Comment:
Based on this update in the README https://github.com/michaelgrund/GMT-plotting/blame/8dfbf9620a343d821d9f4cda76c1e7380d09094f/README.md#L3

> Updates for PyGMT based notebooks are carried out at irregular intervals. The GMT bash scripts are not maintained anymore.

Please indicate if you prefer to reduce this PR to the change in the related Jupyter notebook.

Reply:
Yes would be fine to just adjust the JN and leave the bash script as is.

Reply:
I have reverted the changes in the two bash scripts in commit https://github.com/michaelgrund/GMT-plotting/pull/16/commits/d0187b4be4b9733683c0e273133797d0393ae477. After this, this PR is reduced to only mentioning the alias `lake` for `C` of `Figure.coast`.